nixos/modules/services/continuous-integration/github-runner/service.nix

```diff
@@ -170,6 +170,8 @@ with lib;
     # If running in ephemeral mode, restart the service on-exit (i.e., successful de-registration of the runner)
     # to trigger a fresh registration.
     Restart = if cfg.ephemeral then "on-success" else "no";
+    # If the runner exits with `ReturnCode.RetryableError = 2`, always restart the service
```

Comment:
Maybe add a comment with the reference you mentioned? (But in permalink form)

https://github.com/actions/runner/blob/40ed7f8/src/Runner.Common/Constants.cs#L146

It's always nice to be able to track things like that back to the source. Otherwise LGTM!

Reply:
Done!